packages/merge-on-green/src/merge-logic.ts

```diff
@@ -470,6 +477,36 @@ mergeOnGreen.getReviewsCompleted = async function getReviewsCompleted(
   }
 };
 
+/**
+ * Function dismisses a review for the most secure version of this bot
+ * @param owner of pr (from Watch PR)
+ * @param repo of pr (from Watch PR)
+ * @param pull_number pr number
+ * @param review_id review id of the review to dismiss
+ * @param github unique installation id for each function
+ * @returns an array of Review types
+ */
+mergeOnGreen.dismissReview = async function dismissReview(
```

Comment:
It kinda looks like this is only called from one location.  Is it worth having it's own function?  Or could you just inline the logic?